zip-XXXX.rst

```diff
@@ -0,0 +1,451 @@
+been structured thus far.
+
+Inevitably, I've become interested in the community's proposals for a new dev
+fee, and thought about how the right structure might support work like ours.
+
+I believe the Zcash community has an opportunity to deploy a new incentive
+structure that will attract companies like ours to build and improve Zcash,
+leading to a more resilient network, stronger technology, and wider usage.
+
+The Zcash narrative
+-------------------
+
+We're all here to build a robust, private, decentralized currency. But in the
+dev fee proposals I've seen so far, the idea of a Zcash narrative that
+distinguishes it from the competition is absent.
+
+Of the slew of ZIPs addressing Zcash's future, there's only been one strong
+narrative case -- the idea that Zcash exists purely as a hedge against Bitcoin's
+long-term privacy failure. Put simply, Zcash is "Bitcoin, but private".
+
+Zcash should aim higher. Bitcoin is the only coin that has successfully made a
+store of value argument, which I like to call "worse is better". Don't upgrade
+the network -- the argument goes -- stability is more important than solving
+today's problems.  Bitcoin is chasing the `Lindy
+effect<https://en.wikipedia.org/wiki/Lindy_effect>`, where worse is better, and
+the network becomes stronger every day it survives. That's great for Bitcoin.
+For the rest of us, though, better is better. Zcash *should be better*.
+
+Zcash is known for having the best tech in the space, built by one of the best
+team's in the space. We should lean in to that reputation, nurturing the best
+research and engineering talent to take Zcash to the next level, and leveraging
+a Zcash dev fee as a differentiator to build the world's best private medium of
+exchange.
+
+Principles of cryptocurrency governance
+---------------------------------------
+
+To understand Zcash governance, it's worth reviewing "default" cryptocurrency
+governance. Most proof-of-work chains today have three major governing roles:
+
+1. Miners validate and secure the chain. They do some work to earn a reward.
+   Miners are the first owners of newly minted coins, and are an integral part
+   of network upgrades.
+2. Users buy, hold, and spend the currency. In networks like Bitcoin, they also
+   run full nodes, strengthening network resilience by decentralizing
+   validation.
+3. Developers maintain clients to power and interact with the network. They
+   typically propose network upgrades.
+
+On a chain like Bitcoin, any of these roles can veto a network upgrade.
+
+1. Miners can veto activating a new fork by refusing to build off blocks using
+   new network rules, orphaning a minority effort. They can also attack any fork
+   attempt that doesn't change
+2. Users can veto a fork by refusing to update their full nodes, rejecting
+   blocks as invalid -- as demonstrated in the UASF fiasco resulting from the
+   SegWit2x attempt to force a Bitcoin hardfork. Users can also vote with their
```

Comment:
```suggestion
   Bitcoin miners' refusal to activate SegWit. Users can also vote with their
```

Reply:
Correction: UASF (via BIP-148) was first proposed before Segwit2x was announced, and was in response to the miners' ongoing refusal to activate SegWit.

Reply:
Ofc, good catch!